umh.docs.umh.app/content/en/docs/production-guide/security/firewall-rules.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Firewall Rules"
+menuTitle: "Firewall Rules"
+description: |
+  This page describes how to setup firewall rules for the UMH instances.
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+For closed enterprise networks with specific permissions, this page outlines firewall rule configuration
+to ensure the security and seamless operation of your UMH instances.
+
+## {{% heading "prerequisites" %}}
+
+Before configuring the firewall, let's discuss some key points.
+
+The usage of the components integrated into the UMH require various rules for their proper operation.
+Considering the complexity involved in setting up and maintaining these rules, we have hosted as much
+as possible under our own domain (management.umh.app), which leads to a simpler and less error-prone
+setup.
+
+However, it's important to note that, some components may require occasional communication to external
+services. For example, installing plugins for applications like Grafana or Node-RED will likely trigger
+requests to their respective domains. In such cases, this page provides a comprehensive list of external
+domains and ports that are relevant during the usage of the UMH, helping you to anticipate and configure
+the rules accordingly.
+
+## Firewall Configuration
+
+Once you're ready and ensured that you have the necessary permissions to configure the firewall, follow these steps:
+
+### Whitelist Our Domain
+
+The first and most important step simply involves allowing access to our domain (management.umh.app) on the following port:
+
+- TCP port 443: HTTPS traffic
+
+Ignoring this step or even worse, blocking access to our domain and corresponding port, will result in a malfunctioning
+UMH instance.
+
+### Whitelist External Domains
```

Comment:
Optional: whitelist domains for common 3rd party plugins

Include these common external domains and ports in your firewall rules to allow installing Node-RED and Grafana plugins:
...
Depending on your setup, additional domains may need to be whitelisted.